### PR TITLE
Some minor additions.

### PR DESCRIPTION
**Pull request by michaelkleinhenz**
Adds documentation, removes `test-dev` target.

### CODE REVIEW

**Comment by xcoulon** [comment]
I would suggest that you use the same organisation for the `Makefile` as we do in other repositories, and reuse the https://github.com/codeready-toolchain/host-operator/blob/master/make/help.mk sub-file to provide help (`make help`)

**Comment by michaelkleinhenz** [comment]
@xcoulon done.

File: Makefile [hunk 1]
```diff
@@ -1,3 +1,12 @@
+export GO111MODULE=on
+
+# It's necessary to set this because some environments don't link sh -> bash.
+SHELL := /bin/bash
+
+include ./make/*.mk
+.DEFAULT_GOAL := help
+
+
 #! /usr/bin/make
```
Comment (xcoulon) [inline Makefile]: I guess you can remove all those commented lines, now

File: make/image.mk [hunk 2]
```diff
@@ -0,0 +1,5 @@
+.PHONY: image
+## build docker image
+image:
+	docker build -t kleinhenz/registration-service:0.1 .
```
Comment (xcoulon) [inline make/image.mk:4]: why do you want to use your username as the org?

Comment (michaelkleinhenz) [reply make/image.mk:4]: For now as we don't have a repository yet. We will need more time to find the final location.

File: Makefile (hunk 1 above)
Comment (michaelkleinhenz) [reply Makefile]: Already done. See e264d7d. You were just 30s too early or so :-)

**Review by tinakurian — APPROVED** [review]

**Comment by xcoulon** [comment]
> @xcoulon done.

great, thanks!

**Pull request merged**